test/VSSDK.TestExtension/TestExtensionPackage.cs

```diff
@@ -25,18 +25,12 @@ public sealed class TestExtensionPackage : ToolkitPackage
         protected override async Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
         {
             // Tool windows
-            RunnerWindow.Initialize(this);
-            ThemeWindow.Initialize(this);
-            MultiInstanceWindow.Initialize(this);
+            this.AutoRegisterToolWindows();
```

Comment:
Should this be renamed to `RegisterToolWindows` since it doesn't happen automatically? Same with the commands